src/config.rs

```diff
@@ -116,6 +122,26 @@ impl Config {
                     .help("Number of transactions to keep in for query LRU cache")
                     .default_value("10000")  // should be enough for a small wallet.
             )
+            .arg(
+                Arg::with_name("enable_txstore")
+                    .long("enable-txstore")
+                    .help("Store full raw transactions in database")
+            )
+            .arg(
+                Arg::with_name("enable_blockmeta")
+                    .long("enable-blockmeta")
+                    .help("Store block metadata in database")
+            )
+            .arg(
+                Arg::with_name("enable_blocktxs")
+                    .long("enable-blocktxs")
+                    .help("Store blockhash to txids map in database")
```

Comment:
nit: `to txids map` -> `at the txids map`

Reply:
This is not referencing the addition of `blockhash` to the `t` index (which is now the only non-configurable index change from upstream electrs), its a separate index (under the prefix `X`) that allows looking up all the `txids` in a particular block.

Reply:
Oops... you're definitely right, I misread the description.